src/CodeGen_C.cpp

```diff
@@ -585,6 +585,15 @@ void CodeGen_C::compile(const LoweredFunc &f) {
         stream << ") HALIDE_FUNCTION_ATTRS {\n";
         indent += 1;
 
+        // Emit a local user_context we can pass in all cases, either
+        // aliasing __user_context or nullptr.
+        if (!is_header()) {
+            do_indent();
+            stream << "void * const __ucon = " 
```

Comment:
True, but so is "__user_context", which we were already using. (No biggie to change it.)